modules/joda/src/test/scala/pureconfig/module/joda/JodaSpec.scala

```diff
@@ -0,0 +1,62 @@
+package pureconfig.module.joda
+
+import com.typesafe.config.{ ConfigFactory, ConfigValueFactory }
+import org.joda.time._
+import org.scalacheck.Arbitrary
+import JodaSpec._
+import org.joda.time.format.{ DateTimeFormat, DateTimeFormatter }
+import pureconfig.module.joda.configurable.ConfigurableSuite.{ dateTimeArbitrary, zoneIdArbitrary }
+import pureconfig.BaseSuite
+import pureconfig.syntax._
+
+class JodaSpec extends BaseSuite {
+
+  it should "parse Instant" in forAll {
```

Comment:
This test and the ones below all test the reading part of the `ConfigConvert`, but not the writing part.

`BaseSuite` provides a convenient `checkArbitrary[A]` method that checks that writing an `A` to a config and reading it back should yield an `A` instance equal to the original one. Consider replacing this whole test by a single `checkArbitrary[Instant]` (same thing with the other types).